ui/dialogs.go

```diff
@@ -119,3 +121,107 @@ func (win *Window) AddAccountDiag() {
+						layout.Rigid(func() {
+							layout.UniformInset(unit.Dp(10)).Layout(win.gtx, func() {
+								win.outputs.pageInfo.Layout(win.gtx)
+							})
+						}),
+						layout.Rigid(func() {
+							inset := layout.Inset{
+								Left: unit.Dp(190),
+							}
+							inset.Layout(win.gtx, func() {
+								win.outputs.gotItDiag.Layout(win.gtx, &win.inputs.receiveIcons.gotItDiag)
+							})
+						}),
+					)
+				})
+			}
+			decredmaterial.Modal{}.Layout(win.gtx, selectedDetails)
+		})
+	})
+}
+
+func (win *Window) generateNewAddressDiag() {
+	layout.Flex{}.Layout(win.gtx,
+		layout.Flexed(0.80, func() {
+		}),
+		layout.Flexed(1, func() {
+			inset := layout.Inset{
+				Top: unit.Dp(150),
+			}
+			inset.Layout(win.gtx, func() {
+				win.gtx.Constraints.Width.Min = 40
+				win.gtx.Constraints.Height.Min = 40
+				win.outputs.newAddressDiag.Layout(win.gtx, &win.inputs.receiveIcons.newAddressDiag)
+			})
+		}),
+	)
+}
+
+func (win *Window) selectedAcountDiag() {
+	info := win.walletInfo.Wallets[win.selected]
+	win.outputs.selectedWalletNameLabel.Text = info.Name
+	win.outputs.selectedWalletBalLabel.Text = info.Balance
+
+	account := win.walletInfo.Wallets[win.selected].Accounts[win.selectedAccount]
+	win.outputs.selectedAccountNameLabel.Text = account.Name
+	win.outputs.selectedAccountBalanceLabel.Text = account.SpendableBalance
+
+	win.combined.sel.Layout(win.gtx, func() {
+		layout.Flex{}.Layout(win.gtx,
+			layout.Flexed(0.22, func() {
+			}),
+			layout.Flexed(1, func() {
+				layout.Stack{}.Layout(win.gtx,
+					layout.Stacked(func() {
+						selectedDetails := func() {
+							layout.UniformInset(unit.Dp(10)).Layout(win.gtx, func() {
+								layout.Flex{Axis: layout.Vertical}.Layout(win.gtx,
+									layout.Rigid(func() {
+										layout.Flex{}.Layout(win.gtx,
+											layout.Rigid(func() {
+												layout.Inset{Bottom: unit.Dp(5)}.Layout(win.gtx, func() {
+													win.outputs.selectedAccountNameLabel.Layout(win.gtx)
+												})
+											}),
+											layout.Rigid(func() {
+												layout.Inset{Left: unit.Dp(20)}.Layout(win.gtx, func() {
+													win.outputs.selectedAccountBalanceLabel.Layout(win.gtx)
+												})
+											}),
+										)
+									}),
+									layout.Rigid(func() {
+										layout.Inset{Left: unit.Dp(20)}.Layout(win.gtx, func() {
+											layout.Flex{}.Layout(win.gtx,
+												layout.Rigid(func() {
+													layout.Inset{Bottom: unit.Dp(5)}.Layout(win.gtx, func() {
+														win.outputs.selectedWalletNameLabel.Layout(win.gtx)
+													})
+												}),
+												layout.Rigid(func() {
+													layout.Inset{Left: unit.Dp(22)}.Layout(win.gtx, func() {
+														win.outputs.selectedWalletBalLabel.Layout(win.gtx)
+													})
+												}),
+											)
+										})
+									}),
+								)
+							})
+						}
+						decredmaterial.Card{}.Layout(win.gtx, selectedDetails)
+					}),
+				)
+			}),
+		)
+	})
+}
```

Comment:
I'm not sure where this gets displayed or why it would be necessary to display.

Reply:
should I remove it as it is part of the mockup in issue #7

Reply:
Does it get displayed?

Reply:
from the mobile app yes it is.